search.py

```diff
@@ -251,25 +251,8 @@ def Walkable(wMap, sample_rate, start_point, end_point):
         if not tile:
             print('({}, {}) OUT OF BOUNDS'.format(x, y))
             break
-        elif tile.isObstacle or tile.isBloated:
+        elif tile.isObstacle:
             return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x+tile_size, y+tile_size)).isObstacle:
-            return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x+tile_size, y)).isObstacle:
-            return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x+tile_size, y-tile_size)).isObstacle:
-            return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x, y-tile_size)).isObstacle:
-            return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x, y+tile_size)).isObstacle:
-            return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x-tile_size, y+tile_size)).isObstacle:
-            return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x-tile_size, y)).isObstacle:
-            return False
-        elif wMap.get_tile((x+tile_size, y+tile_size)) is not None and wMap.get_tile((x-tile_size, y-tile_size)).isObstacle:
-            return False
-
```

Comment:
I checked all the tiles around the tile to see if they were obstacles because sometimes the path would go through a bloated tile. But the fix for this was actually somewhere else and I never removed this garbage... so now GOODBYE hacky fix